src/pr2.cc

```diff
@@ -306,50 +309,6 @@ void PR2Wrapper::generate_nondet_operator_mappings() {
     }
 
     PR2.proxy->set_nondet_index_map(*nondet_index_map);
-
-    //TODO
-    //Reinstantiate conditional mask and possibly nondetop2fspas
-    //They were previously global and not instantiated anywhere
-
-
-    // /* Build the data structures required for mapping between the
-    //  * deterministic actions and their non-deterministic equivalents. */
-    // int cur_nondet = 0;
-    // for (auto op : PR2.proxy->get_operators()) {
-
-    //     int nondet_index = -1;
-
-    //     PR2.general.conditional_mask.push_back(new vector<int>());
-    //     PR2.deadend.nondetop2fsaps.push_back(new vector< FSAP* >());
-
-    //     if (PRP.general.nondet_index_mapping.find(op.get_nondet_name()) == PRP.general.nondet_index_mapping.end()) {
-
-    //         nondet_index = cur_nondet;
-    //         PRP.general.nondet_index_mapping[op.get_nondet_name()] = cur_nondet;
-
-    //         PRP.general.nondet_mapping.push_back(new vector<GlobalOperator *>());
-    //         PRP.general.conditional_mask.push_back(new vector<int>());
-
-    //         PRP.deadend.nondetop2fsaps.push_back(new vector< FSAP* >());
-
-    //         cur_nondet++;
-
-    //     } else {
-    //         nondet_index = PRP.general.nondet_index_mapping[op.get_nondet_name()];
-    //     }
-
-    //     op.nondet_index = nondet_index;
-    //     PRP.general.nondet_mapping[nondet_index]->push_back(&op);
-    //     op.nondet_outcome = PRP.general.nondet_mapping[nondet_index]->size() - 1;
-
-    //     for (auto eff : op.get_all_effects()) {
-    //         for (auto cond : eff.conditions) {
-    //             vector<int> *var_list = PRP.general.conditional_mask[nondet_index];
-    //             if (find(var_list->begin(), var_list->end(), cond.var) == var_list->end())
-    //                 PRP.general.conditional_mask[nondet_index]->push_back(cond.var);
-    //         }
-    //     }
-    // }
```

Comment:
Isn't it worth keeping this unworking code around for when we re-create the mask?

Reply:
I still have that code in a local repository need be. I removed it to clean up the code. Conditional mask is put in the same places but I don't know if it has been tested yet.